documentation.md

```diff
@@ -19,11 +19,11 @@ For previous versions of the specification, please see the [Specification Links]
 Meta-schemas
 ------------
 
-The meta-schemas are the schemas which define the JSON Schema and Hyper-Schema formats.
-
+The meta-schemas define the JSON Schema and Hyper-Schema formats using the JSON Schema format.
```

Comment:
I think it is worth trying to improve the sentence/explanation.

How about:

Meta-schemas are self-descriptive schemas; they describe the schema format using the format itself.  The Core/Validation Meta-Schema is a valid JSON Schema.  The Hyper Meta-Schema is a valid JSON Hyper-Schema.

___

I noted that referring to these things using natural language was a bit awkward; I may not have identified the correct names.